src/controller/systemartifact/execution.go

```diff
@@ -0,0 +1,153 @@
+package systemartifact
+
+import (
+	"context"
+	"github.com/goharbor/harbor/src/jobservice/job"
+	"github.com/goharbor/harbor/src/jobservice/logger"
+	"github.com/goharbor/harbor/src/lib/log"
+	"github.com/goharbor/harbor/src/lib/orm"
+	"github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/lib/retry"
+	"github.com/goharbor/harbor/src/pkg/scheduler"
+	"github.com/goharbor/harbor/src/pkg/systemartifact"
+	"github.com/goharbor/harbor/src/pkg/task"
+	"time"
+)
+
+const (
+	VendorTypeSystemArtifactCleanup = "SYSTEM_ARTIFACT_CLEANUP"
+	cronTypeDaily                   = "Daily"
+)
+
+func init() {
+	task.SetExecutionSweeperCount(job.SystemArtifactCleanup, 50)
```

Comment:
should be the vendor type?

Reply:
Done. Incidentally, they had the same values